### ISSUE TITLE
Internal server error

### ISSUE THREAD

**Issue by doender** [ParsePlatform/parse-server#101]
I migrated my Parse data over to a local instance. Most queries seem to work, but this one does not:

```
curl 
-H "X-Parse-Application-Id: myAppId"   
-H "Content-Type: application/json"   
http://localhost:1337/parse/classes/Test 
--data-urlencode 'where={"name":{"$exists":true}}'
```

I get: 
```
{"code":1,"message":"Internal server error."}
```

The server log shows:
```
Uncaught internal server error. { [Error: invalid json]
  body: 'where=%7B%22name%22%3A%7B%22%24exists%22%3Atrue%7D%7D',
  status: 400 } Error: invalid json
    at parse (/vagrant/parse-server-example/node_modules/body-parser/lib/types/json.js:79:15)
    at /vagrant/parse-server-example/node_modules/body-parser/lib/read.js:102:18
    at done (/vagrant/parse-server-example/node_modules/raw-body/index.js:248:14)
    at IncomingMessage.onEnd (/vagrant/parse-server-example/node_modules/raw-body/index.js:294:7)
    at IncomingMessage.g (events.js:273:16)
    at emitNone (events.js:80:13)
    at IncomingMessage.emit (events.js:179:7)
    at endReadableNT (_stream_readable.js:906:12)
    at nextTickCallbackWith2Args (node.js:474:9)
    at process._tickCallback (node.js:388:17)
```

**Comment by gfosco** [comment]
Ok, looks like another bug related to query strings... Will look in to it.

**Comment by nlutsenko** [comment]
Fixed with #543.
